tests/org.jboss.tools.forge.ui.bot.test/META-INF/MANIFEST.MF

```diff
@@ -8,7 +8,7 @@ Bundle-RequiredExecutionEnvironment: JavaSE-1.7
 Require-Bundle: org.jboss.tools.forge.ui;bundle-version="1.0.0",
  org.jboss.tools.forge.core;bundle-version="1.0.0",
  org.jboss.tools.forge.runtime;bundle-version="1.0.0",
- org.jboss.tools.forge.reddeer;bundle-version="[4.4.2,4.5.0]",
+ org.jboss.tools.forge.reddeer;bundle-version="2.0.2",

```

Comment:
Since the version of moved plugins is lover than the current one, you need to use range properly. Otherwise cached 4.5.x version will be used. Use e.g. [2.0.201,3.0.0)